prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -204,28 +204,12 @@ def prompt_processor() -> Any:
     for output in outputs:  # type:ignore
         variable_names.append(output[PSKeys.NAME])
     for output in outputs:  # type:ignore
-        is_active = output[PSKeys.ACTIVE]
         prompt_name = output[PSKeys.NAME]
         promptx = output[PSKeys.PROMPT]
         chunk_size = output[PSKeys.CHUNK_SIZE]
         util = PromptServiceBaseTool(log_level=LogLevel.INFO, platform_key=platform_key)
         index = Index(tool=util)
 
-        if is_active is False:
```

Comment:
@jagadeeswaran-zipstack why did we remove this check?  Ideally inactive prompts should be skipped only right?

Reply:
We have this check in backend now. Inactive prompts cannot be exported by the user.